Patch by Zarafa <http://www.zarafa.com/> - headers can be broken over multiple lines in an email. When
the wrapped line directly started with quoted-printable, vmime copied this text as normal text, instead
of parsing it as quoted-printable. 

--- libvmime-0.7.1/src/word.cpp						2007-01-26 14:44:10.251563000 +0100
+++ libvmime-0.7.1/src/word.cpp.qp-starts-on-second-line		2007-01-26 14:45:01.074739500 +0100
@@ -91,6 +91,7 @@
 			unencoded += ' ';
 
 			startPos = pos;
+			continue;
 		}
 		// Start of an encoded word
 		else if (pos + 8 < end &&  // 8 = "=?(.+)?(.+)?(.*)?="
